### DISCUSSION TITLE
Latex rendering size control?

### DISCUSSION THREAD

**Discussion by NelkonParker** [xournalpp/xournalpp#3858] [Q&A] [answered]
How do I change the default size at which latex is rendered? I would like to increase it, but I'm not at all sure what it is controlled by. Presumably something in default_template.tex?

**Comment by tattsan** [comment, marked as answer]
`default_template.tex` uses `standalone.cls`, and this class uses `article.cls` as its base class file. `article.cls` receives `10pt` or `11pt` or `12pt` as its font size option, and the default is `10pt`.  So you can change this to `12pt` like this:
```
\documentclass[varwidth=true, crop, border=5pt, 12pt]{standalone}
```
As discussed in #3293, you may remove `varwidth=true` .
If you want bigger size, you can change the base class file of `standalone.cls` as follows:
```
\documentclass[varwidth=true, crop, border=5pt, class=scrartcl, fontsize=40pt]{standalone}
\usepackage{lmodern}
```
where `scrartcl.cls` is a class file in `KOMA-Script` package bundle. Some TeX distributions may not include `KOMA-Script`, so you may want to check to see if you have `scrextend.sty` or `scrartcl.cls` installed.
You need a scalable font package like `lmodern` to prevent fontsize substitution of computer modern fonts.  If you don't want to change the base class file for `standalone.cls`, you can use `scrextend` package:
```
\documentclass[varwidth=true, crop, border=5pt]{standalone}
\usepackage{lmodern}
\usepackage{scrextend}
\changefontsizes{80pt}
```
If you want to change fontsize in `LaTeX` code, the following macro is useful (  `KOMA-Script` is required ) :
```
\newcommand*\myFsize[1]{\)\changefontsizes{#1}\(\displaystyle }
```

**Reply by NelkonParker** [reply]
Thanks. This does the job. It would be nice to have real time control over this though, rather than having to edit a template file. Still, I can live with it for now.

**Reply by tattsan** [reply]
You can use `\changefontsize`  in your `LaTeX` code if you are using `scrartcl.cls` or `scrextend.sty` . Because it should not be in math mode, your `LaTeX` code should be like this:
```
\)\changefontsizes{100pt}\(\displaystyle \int x^2\,dx
```
This requires `\displaystyle` because it exits math mode once. If you feel this tedious, you can make it a macro:
```
\newcommand*\myFsize[1]{\)\changefontsizes{#1}\(\displaystyle }
```
